sky/packages/sky/lib/src/animation/animated_value.dart

```diff
@@ -25,70 +25,43 @@ abstract class Animatable {
   String toString();
 }
 
-/// Used by [AnimationPerformance] to convert the timing of a performance to a different timescale.
+/// Used by [Performance] to convert the timing of a performance to a different timescale.
 /// For example, by setting different values for the interval and reverseInterval, a performance
 /// can be made to take longer in one direction that the other.
 class AnimationTiming {
-  AnimationTiming({
-    this.interval: const Interval(0.0, 1.0),
-    this.reverseInterval,
-    this.curve: linear,
-    this.reverseCurve
-  });
+  AnimationTiming({ this.curve, this.reverseCurve });
 
-  /// The interval during which this timing is active in the forward direction
-  Interval interval;
-
-  /// The interval during which this timing is active in the reverse direction
-  ///
-  /// If this field is null, the timing defaults to using [interval] in both directions.
-  Interval reverseInterval;
-
-  /// The curve that this timing applies to the animation clock in the forward direction
+  /// The curve to use in the forward direction
```

Comment:
(nit: add periods at the end of dart docs when you come across those that don't have them. don't worry about doing it in this PR necessarily)